skyportal/models.py

```diff
@@ -1700,6 +1729,20 @@ class Photometry(Base, ha.Point):
     )
     assignment = relationship('ClassicalAssignment', back_populates='photometry')
 
+    owner_id = sa.Column(
+        sa.ForeignKey('users.id', ondelete='CASCADE'),
+        nullable=True,
```

Comment:
Why not False? There's always an associated token/user, right?

Reply:
Fair enough

Reply:
This will make the migration a bit harder though - what should we use for data that are already in there ?

Reply:
hmm. for fritz.science let's do id=3 (kowalski-bot)

Reply:
i did id=1 (provisioned admin) as id=3 is not always defined on SP